projects/clas12MC.yaml

```
@@ -0,0 +1,11 @@
+Name: clas12MC
```

```suggestion
Name: JLab_clas12MC
```

Perhaps? @lmichael107

---

This is for an existing project which was not registered